src/store/index.js

```diff
@@ -14,7 +14,7 @@ import transactions, {
   TRANSACTIONS_REDUCER_ID,
 } from '~/routes/safe/store/reducer/transactions'
 
-export const history = createBrowserHistory()
+export const history = process.env.NODE_ENV === 'test' ? createMemoryHistory() : createBrowserHistory()
```

Comment:
createMemoryHistory doesn't work for us, because we check `window.location` in tests and memoryHistory was created for ENVs without window